src/spmat/linalg.pyx

```diff
@@ -8,15 +8,26 @@ cimport scipy.linalg.cython_lapack as lapack
 @cython.boundscheck(False)
 @cython.wraparound(False)
 def block_lsvd(double[:, ::1] a_view, long[::1] n_view, long[::1] k_view):
-    cdef int dim_row = max(n_view)
```

Comment:
I just left these annotations in for my own purposes, when trying to understand the Cython code, and forgot to delete. Happy to leave them or delete them, up to you all